# Generated by the protocol buffer compiler.  DO NOT EDIT!
# source: APIException.proto

import sys
_b=sys.version_info[0]<3 and (lambda x:x) or (lambda x:x.encode('latin1'))
from google.protobuf import descriptor as _descriptor
from google.protobuf import message as _message
from google.protobuf import reflection as _reflection
from google.protobuf import symbol_database as _symbol_database
from google.protobuf import descriptor_pb2
# @@protoc_insertion_point(imports)

_sym_db = _symbol_database.Default()


from IDL.Protos.Extensions import CustomOptions_pb2 as IDL_dot_Protos_dot_Extensions_dot_CustomOptions__pb2


DESCRIPTOR = _descriptor.FileDescriptor(
    name='APIException.proto',
    package='da.core',
    syntax='proto2',
    serialized_pb=_b('\n\x12\x41PIException.proto\x12\x07\x64\x61.core\x1a)IDL/Protos/Extensions/CustomOptions.proto\"m\n\x0c\x44vxException\x12.\n\x08\x65rrorMsg\x18\x01 \x01(\tB\x12\x92?\x0f\x32\rError messageR\x08\x65rrorMsg\x12-\n\terrorCode\x18\x02 \x01(\rB\x0f\x92?\x0c\x32\nError codeR\terrorCode')
    ,
    dependencies=[IDL_dot_Protos_dot_Extensions_dot_CustomOptions__pb2.DESCRIPTOR,])
_sym_db.RegisterFileDescriptor(DESCRIPTOR)




_DVXEXCEPTION = _descriptor.Descriptor(
    name='DvxException',
    full_name='da.core.DvxException',
    filename=None,
    file=DESCRIPTOR,
    containing_type=None,
    fields=[
        _descriptor.FieldDescriptor(
            name='errorMsg', full_name='da.core.DvxException.errorMsg', index=0,
            number=1, type=9, cpp_type=9, label=1,
            has_default_value=False, default_value=_b("").decode('utf-8'),
            message_type=None, enum_type=None, containing_type=None,
            is_extension=False, extension_scope=None,
            options=_descriptor._ParseOptions(descriptor_pb2.FieldOptions(), _b('\222?\0172\rError message'))),
        _descriptor.FieldDescriptor(
            name='errorCode', full_name='da.core.DvxException.errorCode', index=1,
            number=2, type=13, cpp_type=3, label=1,
            has_default_value=False, default_value=0,
            message_type=None, enum_type=None, containing_type=None,
            is_extension=False, extension_scope=None,
            options=_descriptor._ParseOptions(descriptor_pb2.FieldOptions(), _b('\222?\0142\nError code'))),
    ],
    extensions=[
    ],
    nested_types=[],
    enum_types=[
    ],
    options=None,
    is_extendable=False,
    syntax='proto2',
    extension_ranges=[],
    oneofs=[
    ],
    serialized_start=74,
    serialized_end=183,
)

DESCRIPTOR.message_types_by_name['DvxException'] = _DVXEXCEPTION

DvxException = _reflection.GeneratedProtocolMessageType('DvxException', (_message.Message,), dict(
    DESCRIPTOR = _DVXEXCEPTION,
    __module__ = 'APIException_pb2'
    # @@protoc_insertion_point(class_scope:da.core.DvxException)
    ))
_sym_db.RegisterMessage(DvxException)


_DVXEXCEPTION.fields_by_name['errorMsg'].has_options = True
_DVXEXCEPTION.fields_by_name['errorMsg']._options = _descriptor._ParseOptions(descriptor_pb2.FieldOptions(), _b('\222?\0172\rError message'))
_DVXEXCEPTION.fields_by_name['errorCode'].has_options = True
_DVXEXCEPTION.fields_by_name['errorCode']._options = _descriptor._ParseOptions(descriptor_pb2.FieldOptions(), _b('\222?\0142\nError code'))
# @@protoc_insertion_point(module_scope)
